README.md

```diff
@@ -50,6 +53,8 @@ You can use a custom template, in that case plugin will pass the following varia
 | `reading_time` | Reading time of the article |
 | `preview_picture` | link to preview picture of the article |
 | `domain_name` | Link to the source domain article |
+| `is_archived` | Whether the article is archived or not |
+| `is_starred` | Whether the article is starred or not |
```

Comment:
I can imagine this 'drifting' from reality once someone has changed the read/starred status of an article in Wallabag. Subsequent sync won't update these values right? Maybe good to clearly mention that in the docs.

Reply:
Which would also mean that when people let articles be synced into specific folders drift could occur once they change read/starred state in Wallabag.